test/boost/database_test.cc

```diff
@@ -287,12 +291,20 @@ static void test_database(void (*run_tests)(populate_fn_ex, bool)) {
     }).get();
 }
 
-SEASTAR_THREAD_TEST_CASE(test_database_with_data_in_sstables_is_a_mutation_source_plain) {
-    test_database(run_mutation_source_tests_plain);
+SEASTAR_THREAD_TEST_CASE(test_database_with_data_in_sstables_is_a_mutation_source_plain_cg0) {
+    test_database(run_mutation_source_tests_plain, 0);
+}
+
+SEASTAR_THREAD_TEST_CASE(test_database_with_data_in_sstables_is_a_mutation_source_plain_cg1) {
+    test_database(run_mutation_source_tests_plain, 1);
+}
+
+SEASTAR_THREAD_TEST_CASE(test_database_with_data_in_sstables_is_a_mutation_source_reverse_cg0) {
+    test_database(run_mutation_source_tests_reverse, 0);
 }
 
-SEASTAR_THREAD_TEST_CASE(test_database_with_data_in_sstables_is_a_mutation_source_reverse) {
-    test_database(run_mutation_source_tests_reverse);
+SEASTAR_THREAD_TEST_CASE(test_database_with_data_in_sstables_is_a_mutation_source_reverse_cg1) {
+    test_database(run_mutation_source_tests_reverse, 1);
```

Comment:
We should consider reducing the amount of data. I don't know it this is really testing anything important.

Reply:
Perhaps we should just pick one test or two for stressing multiple groups, to avoid the poor me from having to fix regressions in the area constantly. In order to stress the ability to write into / read from multiple groups. And then set up a weekly CI test which runs the entire suite with multiple groups.